src/cbl-mariner/2.0/webassembly/amd64/Dockerfile

```diff
@@ -6,6 +6,7 @@ RUN tdnf update -y \
         # WebAssembly build dependencies
         which \
         nodejs \
+        npm \
```

Comment:
we shouldn't need to change the mariner 2.0 images, only 3.0 was affected afaik

Reply:
I think there is a confusion. nodejs used to install npm transitively, now it doesn't. The next time this image will be built we will be missing npm. The failing build didn't built cbl images. I updated it "while I was at it" so the next person don't run into it.

Reply:
that's the case for 3.0, but in 2.0 the nodejs package still provides the npm command:

```
$ docker run --rm mcr.microsoft.com/cbl-mariner/base/core:2.0 sh -c "tdnf install -y nodejs && npm --help"
Loaded plugin: tdnfrepogpgcheck
Refreshing metadata for: 'CBL-Mariner Official Microsoft 2.0 aarch64'
Refreshing metadata for: 'CBL-Mariner Official Extras 2.0 aarch64'
Refreshing metadata for: 'CBL-Mariner Official Base 2.0 aarch64'

Installing:
c-ares           aarch64      1.19.1-1.cm2     mariner-official-base 278.56k   121.55k
brotli           aarch64      1.0.9-2.cm2      mariner-official-base   1.48M   738.43k
nodejs18         aarch64      18.20.2-2.cm2    mariner-official-base  54.95M    16.43M

Total installed size:  56.70M
Total download size:  17.27M

importing key from file:///etc/pki/rpm-gpg/MICROSOFT-RPM-GPG-KEY
importing key from file:///etc/pki/rpm-gpg/MICROSOFT-METADATA-GPG-KEY


Testing transaction
Running transaction
Installing/Updating: brotli-1.0.9-2.cm2.aarch64
Installing/Updating: c-ares-1.19.1-1.cm2.aarch64
Installing/Updating: nodejs18-18.20.2-2.cm2.aarch64
npm <command>

Usage:

npm install        install all the dependencies in your project
npm install <foo>  add the <foo> dependency to your project
npm test           run this project's tests
npm run <foo>      run the script named <foo>
npm <command> -h   quick help on <command>
npm -l             display usage info for all commands
npm help <term>    search for help on <term>
npm help npm       more involved overview

All commands:

    access, adduser, audit, bugs, cache, ci, completion,
    config, dedupe, deprecate, diff, dist-tag, docs, doctor,
    edit, exec, explain, explore, find-dupes, fund, get, help,
    help-search, hook, init, install, install-ci-test,
    install-test, link, ll, login, logout, ls, org, outdated,
    owner, pack, ping, pkg, prefix, profile, prune, publish,
    query, rebuild, repo, restart, root, run-script, sbom,
    search, set, shrinkwrap, star, stars, start, stop, team,
    test, token, uninstall, unpublish, unstar, update, version,
    view, whoami

Specify configs in the ini-formatted file:
    /root/.npmrc
or on the command line via: npm <command> --key=value

More configuration info: npm help config
Configuration fields: npm help 7 config

npm@10.5.0 /usr/lib/node_modules/npm
```

Reply:
I'd prefer not to touch 2.0 if we can avoid it.

Reply:
I've removed the change since the PR looks good otherwise.

Reply:
Alrighty then.I don't mind either way. Being explicit vs. relying on transitive dependencies. 👍